vscode-dotnet-runtime-extension/src/test/functional/DotnetCoreAcquisitionExtension.test.ts

```diff
@@ -379,6 +379,14 @@ suite('DotnetCoreAcquisitionExtension End to End', function()
     }
   }).timeout(standardTimeoutTime);
 
+  test('Find dotnet PATH Command Unmet Runtime Patch Condition', async () => {
+    // Install 8.0.{LATEST, which will be < 99}, look for 8.0.99 with accepting dotnet gr than or eq to 8.0.99
+    // No tests for SDK since that's harder to replicate with a global install and different machine states
+    await findPathWithRequirementAndInstall('8.0', 'runtime', os.arch(), 'greater_than_or_equal', true,

```

Comment:
This may actually need to be false

Reply:
I did look at this line specifically during my review because the comment text confused me a bit. The boolean goes to `shouldFind`, right?

Reply:
Ya, thanks for looking at that. That's right. There are so many parameters going into this test helper function, the way I wrote it makes it kind of mind bending to tell what its doing 😵